app/views/people/_current_roles.html.erb

```diff
@@ -1,22 +1,27 @@
-<section id="current-roles">
-  <% person.current_roles.each do |role| %>
-    <section class="role_appointment role govuk-!-padding-bottom-8">
-    <%= render "govuk_publishing_components/components/heading", {
-        text: person.current_roles_title,
-        id: person.current_roles_title.parameterize,
-        margin_bottom: 2,
-    } %>
-    <%= render "govuk_publishing_components/components/govspeak", {
-    } do %>
-      <%= role["details"]["body"].html_safe %>
-    <% end  %>
+<% if person.currently_in_a_role? %>
+  <section id="current-roles">
+    <% person.current_roles.each do |role| %>
+      <section class="role_appointment role govuk-!-padding-bottom-8">
+        <%= render "govuk_publishing_components/components/heading", {
+            text: person.current_roles_title,
+            id: person.current_roles_title.parameterize,
+            margin_bottom: 2,
+        } %>
 
-    <% if role["document_type"] == "ministerial_role" %>
-      <p class="govuk-body-s"><a href=<%= role["base_path"] %>>More about this role</a></p>
-    <% end %>
+        <%= render "govuk_publishing_components/components/govspeak", {
+        } do %>
+          <%= role["details"]["body"].html_safe %>
+        <% end  %>
+
+        <% if role["document_type"] == "ministerial_role" %>
+          <p class="govuk-body-s"><a href=<%= role["base_path"] %>>More about this role</a></p>
+        <% end %>
 
-    <% parent_organisations = role["links"]["ordered_parent_organisations"].map { |parent| link_to(parent["title"], parent["base_path"]) }%>
-    <%= parent_organisations.to_sentence.html_safe %>
-    </section>
-  <% end %>
-</section>
+        <%= role["links"].fetch("ordered_parent_organisations", [])
```

Comment:
This may need the `govuk-body` class on it (or it's parent section) so that it gets our usual font styling and matches other text on the page.

It'll also need `govuk-link` to get the GOVUK Frontend hover, visited and focus styles.

Reply:
Thanks! I've pushed up a commit which fixes this.